examples/ads.amp.html

```diff
@@ -622,7 +622,7 @@ <h2 id="custom">Custom leaderboard</h2>
       data-slot="1"
       data-url="/examples/custom.ad.example.json">
     <template type="amp-mustache" id="amp-template-id-leader">
-      <a href="{{href}}" target='_blank'>
+      <a href="{{href}}" target='_blank' rel="noopener noreferrer">
```

Comment:
OOC why are these problematic?

Reply:
It's a security risk. See https://lgtm.com/rules/1806963085/